docker/Dockerfile

```diff
@@ -77,6 +77,9 @@ RUN ln -s /var/lib/runit/service/nginx /etc/service/nginx
 ADD dist /opt/spark/dist
 ADD krb5.conf.mustache /etc/
 
+# Add external jar for pushing metrics via StatsD
+ADD jars/spark-statsd-1.0.0.jar /opt/spark/dist/jars/
```

Comment:
@hectorj2f How did you build this jar? Is it from https://github.com/vidhyaarvind?